pkgs/os-specific/darwin/xcode/sdk-pkgs.nix

```diff
@@ -51,9 +49,9 @@ rec {
       echo "-target ${targetPlatform.config} -arch ${iosPlatformArch targetPlatform}" >> $out/nix-support/cc-cflags
       echo "-isystem ${sdk}/usr/include${lib.optionalString (lib.versionAtLeast "10" sdk.version) " -isystem ${sdk}/usr/include/c++/4.2.1/ -stdlib=libstdc++"}" >> $out/nix-support/cc-cflags
     '' + stdenv.lib.optionalString (sdk.platform == "iPhoneSimulator") ''
-      echo "-mios-simulator-version-min=${minSdkVersion}" >> $out/nix-support/cc-cflags
+      echo "-Wl,-platform_version,ios-sim,${targetPlatform.minSdkVersion or "9.0"},${sdk.version}" >> $out/nix-support/cc-cflags
     '' + stdenv.lib.optionalString (sdk.platform == "iPhoneOS") ''
-      echo "-miphoneos-version-min=${minSdkVersion}" >> $out/nix-support/cc-cflags
+      echo "-Wl,-platform_version,ios,${targetPlatform.minSdkVersion or "9.0"},${sdk.version}" >> $out/nix-support/cc-cflags
     '';
```

Comment:
Maybe we should move these to the bintuils override above?

Reply:
Yeah - that avoids the warning

Reply:
I think the c compiler should know about minSdkVersion, so I readded miphoneos-version-min.

Reply:
Weird it knows about it, but can't also send it to the linker, ugh.

Reply:
It sends the min sdk version to the linker, but not the sdk version. It looks like this might fix that:

https://reviews.llvm.org/D71579